data/binding/gen.go

```diff
@@ -582,7 +605,28 @@ type boundExternal{{ .Name }}ListItem struct {
 	old {{ .Type }}
 }
 
+{{- if eq .Name "Untyped" }}
+func (b *boundExternal{{ .Name }}ListItem) setIfChanged(val {{ .Type }}, comparator UntypedItemComparator) error {
```

Comment:
Is this needed? Isn't the comparator internal state anyway? Instead of passing can't we just access the struct field?